### ISSUE TITLE
Show BaseModelica simulation output in OMEdit as if it came from the original Modelica model

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#12322]
After #12318, the simulation results of flat Base Modelica models will show the quoted identifiers at one single level. This reflects the structure of the Base Modelica model (which has no structure) but it may make exploring the model tricky when it comes from a large, structured Modelica model.

In that case, it would be nice to have one option that strips the quotes from the quoted identifiers and uses the dots to populate the usual tree structure of the Variable Browser.

Here is an MWE for that

```modelica
package P
  model M
    Real x = 1;
  end M;
  model S
    M m1(x = time);
    M m2(x = 10);
  end S;
end P;
```
The Base Modelica version of P.S is
```modelica
package 'S'
  model 'S'
    Real 'm1.x' = time;
    Real 'm2.x' = 10.0;
  end 'S';
end 'S';
```
When the flat `S.S` is simulated, it should be possible to optionally view the results of the simulation as the ones of the structure `P.S`
